website/public/locales/hu/message.json

```diff
@@ -1,20 +1,23 @@
 {
-  "copy_message_id": "Copy message ID",
-  "label_action": "Címkézés",
+  "copy_message_id": "Üzenetazonosító másolása",
```

Comment:
ide csak azért kerüljön be az ID, hogy biztosan megtalálják az adminok, akik nyelvet váltanak (pl: Üzenetazonosító IID másolása)

Reply:
We can ignore this for now. This is admin only feature. And translation probably will be removed soon.